fili-core/src/main/java/com/yahoo/bard/webservice/web/SlicesApiRequest.java

```diff
@@ -165,7 +164,7 @@ protected SlicesApiRequest() {
             throw new BadApiRequestException(msg);
         }
 
-        Map<Column, List<Interval>> columnCache = table.getAvailability().getAvailableIntervals();
+        Map<Column, Set<Interval>> columnCache = table.getAllAvailableIntervals();
```

Comment:
Not sure how the ordering of interval is used in the response, but I do want to argue changing the core code just for less painful tests is not worth it, and it would be great if the actual case that make this hard to test could be pointed out so that I can check if it make sense to make it a list just for testing.